File: sbe/binance/spot_3_2/generated-python/spot_sbe/ListStatusType.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple
from io import BytesIO

class ListStatusType:
    """Enum values."""
    RESPONSE = 0
    EXEC_STARTED = 1
    ALL_DONE = 2
    UPDATED = 3
    NON_REPRESENTABLE = 254
    NULL_VALUE = 255

    @staticmethod
    def encoded_length() -> int:
        return 1

File: sbe/binance/spot_3_1/generated-php/spot_sbe/ListStatusType.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class ListStatusType
{
    public const RESPONSE = 0;
    public const EXECSTARTED = 1;
    public const ALLDONE = 2;
    public const UPDATED = 3;
    public const NONREPRESENTABLE = 254;
    public const NULL_VALUE = 255;

    public static function encodedLength(): int
    {
        return 1;
    }
}
